src/DataTable/Table.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Row from "./Row";
+const Table = ({ rowsFound, currentPageNumber, rowsPerPage }) => (
```

Comment:
I think `DataTable.jsx` is tooooo big,  I want to separate some parts out, this is one of them